packages/ui/src/components/sheet.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react'
+import { Sheet as TamaguiSheet } from 'tamagui'
+
+import { Title1 } from './typographies/title1'
+
+interface SheetProps {
+    children?: string
+    open?: boolean
+    title?: string
+  }
+
+/**
+ *
+ */
+export function Sheet({ children, open: _open = true, title }: SheetProps) {
```

Comment:
Maybe I'm mistaken, but I didn't see any usage of this. Why did you add this? If it is not used, remove it please.